src/validate.ts

```diff
@@ -1,7 +1,7 @@
 import * as vscode from "vscode";
 import axios from "axios";
 
-const ENDPOINT = "https://api.codecov.io/validate/v2";
+const ENDPOINT = "https://api.codecov.io/validate/v2?source=vscode_extension";
```

Comment:
preference thing but maybe we could do something a little more shorter like just vsc

Reply:
agree